pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -613,20 +612,24 @@ func processEvents(image string, logChan <-chan logEntry,
 			log.Debugf("processEvents(%s): sending at messageCount %d, byteCount %d",
 				image, messageCount, byteCount)
 			if event.isAppLog {
-				sent, is4xx = sendProtoStrForAppLogs(appUUID, appLogBundle, iteration, image)
+				var appSendTime uint64
+				sent, is4xx, appSendTime = sendProtoStrForAppLogs(appUUID, appLogBundle, iteration, image)
 				if is4xx {
 					logMetrics.Num4xxResponses += uint64(messageCount)
 				} else {
 					logMetrics.NumAppEventsSent += uint64(messageCount)
 					logMetrics.NumAppBundleProtoBytesSent += uint64(byteCount)
+					logMetrics.AvgAppBundleSendTime = (logMetrics.AvgAppBundleSendTime*logMetrics.NumAppBundlesSent + appSendTime) / (logMetrics.NumAppBundlesSent + 1)
 					logMetrics.NumAppBundlesSent++
 					logMetrics.LastAppBundleSendTime = time.Now()
 				}
 				delete(appLogBundles, appUUID)
 			} else {
-				sent = sendProtoStrForLogs(reportLogs, image, iteration, eveVersion)
+				var devSendTime uint64
+				sent, devSendTime = sendProtoStrForLogs(reportLogs, image, iteration, eveVersion)
 				logMetrics.NumDeviceEventsSent += uint64(messageCount)
 				logMetrics.NumDeviceBundleProtoBytesSent += uint64(byteCount)
+				logMetrics.AvgDeviceBundleSendTime = (logMetrics.AvgDeviceBundleSendTime*logMetrics.NumDeviceBundlesSent + devSendTime) / (logMetrics.NumDeviceBundlesSent + 1)
```

Comment:
1. Do we still need this as part of LogMetrics given that we have it in zedcloudMetrics?
2. Why re-compute an average every time? Tracking the sum and count is sufficient.